ui/app/templates/components/pki/config-pki.hbs

```diff
@@ -16,10 +16,29 @@
 
 <MessageError @model={{this.config}} @errors={{this.errors}} />
 <form {{action "save" this.section on="submit"}} class="box is-shadowless is-marginless is-fullwidth has-slim-padding">
-  {{#each (get this.config (concat this.section "Attrs")) as |attr|}}
-    <FormField @attr={{attr}} @model={{this.config}} @data-test-field={{attr.name}} />
-  {{/each}}
-  <div class="field is-grouped box is-fullwidth is-bottomless">
+  {{#let (get this.config (concat this.section "Attrs")) as |sectionAttrs|}}
```

Comment:
This #let feels a little wonky to grab the first attribute in the array (expiry) but since we're renovating this section of the codebase soon, I went for what felt like the most straightforward solution working with what exists.

The pki-config adapter relies on each attribute to be a field so it updates correctly. So disable has to be included as a key on crlAttrs, but instead of rendering a FormField for it, we set it by the value of buildCrl

Open to suggestions on making this cleaner!

Reply:
I think in other forms we've checked inside the `{{#each}}` to see if the attr.name matches the special case (in this case `expiry`) and then render what we need to, and otherwise render regularly. 

I think calculating the value of `disable` based on `crlAttrs` is totally reasonable. And the {{let}} seems fine too, just wanted to share what we've done in the past